src/test/java/base/BaseTest.java

```diff
@@ -1,71 +1,71 @@
 package base;
 
-import static java.util.Optional.ofNullable;
-import static utils.PropertiesHelper.getInstance;
-
 import manager.PageFactoryManager;
 import org.apache.log4j.PropertyConfigurator;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.openqa.selenium.WebDriver;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Listeners;
-import org.testng.annotations.Optional;
-import org.testng.annotations.Parameters;
 import pages.HomePage;
 import utils.PropertiesHelper;
-import reporting.TestListener;
 import webDriverFactory.LocalWebDriverCreator;
 import webDriverFactory.RemoteWebDriverCreator;
 import webDriverFactory.WebDriverCreator;
 
-@Listeners({TestListener.class})
+import static java.util.Optional.ofNullable;
+import static utils.PropertiesHelper.getInstance;
+
 public class BaseTest {
```

Comment:
In general, the `BaseTest` class is overloaded and contains a lot of different logic. It violates the Single Responsibility Principle. All logic that is not related to the main responsibilities should be moved to separate classes. For example, constant configuration and driver configuration logic.